---
layout: post
title:  "Hammerhead"
date:   2022-04-18T14:25:52-05:00
author: Huang Yuanyin, Lin Xiaowei
permalink: /hammer/
---

**Rfam ID: [RF00008](https://rfam.xfam.org/family/RF00008)** <br>
&emsp;&emsp;&emsp;&emsp;**[RF00163](https://rfam.xfam.org/family/RF00163)** <br>
&emsp;&emsp;&emsp;&emsp;**[RF02275](https://rfam.xfam.org/family/RF02275)** <br>
&emsp;&emsp;&emsp;&emsp;**[RF02276](https://rfam.xfam.org/family/RF02276)** <br>
&emsp;&emsp;&emsp;&emsp;**[RF02277](https://rfam.xfam.org/family/RF02277)** <br>
&emsp;&emsp;&emsp;&emsp;**[RF03152](https://rfam.xfam.org/family/RF03152)** <br>


#### click into different sections:

- [Timeline](#timeline)
- [Description](#description)
- [Structure](#structure)
- [References](#references)




## Timeline

***
<html lang="zh-cn">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>

</head>
<style>
   table {
        border: 2px solid #f8f8ff;
        border: 2px solid #767676;
		    border: 2px solid #767676;
		    border-radius: 5px;
		    background-color: #fff;
        }
.timeline {
  list-style: none;
  padding: 10px 0 10px;
  position: relative;
}
.timeline:before {
  top: 0;
  bottom: 0;
  position: absolute;
  content: " ";
  width: 3px;
  background-color: #eeeeee;
  left: 50%;
  margin-left: -1.5px;
}
.timeline > li {
  margin-bottom: 10px;
  position: relative;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li > .timeline-panel {
  width: 46%;
  float: left;
  border: 1px solid #d4d4d4;
  border-radius: 2px;
  padding: 5px;
  position: relative;
  -webkit-box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
}
.timeline > li > .timeline-panel:before {
  position: absolute;
  top: 16px;
  right: -12px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #ccc;
  border-right: 0 solid #ccc;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-panel:after {
  position: absolute;
  top: 16px;
  right: -10px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #fff;
  border-right: 0 solid #fff;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-badge {
  color: #fff;
  width: 25px;
  height: 25px;
  line-height: 40px;
  font-size: 1.4em;
  text-align: center;
  position: absolute;
  top: 16px;
  left: 48.5%;
  margin-left: 0px;
  background-color: #999999;
  z-index: 100;
  border-top-right-radius: 50%;
  border-top-left-radius: 50%;
  border-bottom-right-radius: 50%;
  border-bottom-left-radius: 50%;
}
.timeline > li.timeline-inverted > .timeline-panel {
  float: right;
}
.timeline > li.timeline-inverted > .timeline-panel:before {
  border-left-width: 0;
  border-right-width: 15px;
  left: -15px;
  right: auto;
}
.timeline > li.timeline-inverted > .timeline-panel:after {
  border-left-width: 0;
  border-right-width: 14px;
  left: -14px;
  right: auto;
}
.timeline-badge.primary {
  background-color: #2e6da4 !important;
}
.timeline-badge.success {
  background-color: #3f903f !important;
}
.timeline-badge.warning {
  background-color: #f0ad4e !important;
}
.timeline-badge.danger {
  background-color: #d9534f !important;
}
.timeline-badge.info {
  background-color: #5bc0de !important;
}
.timeline-title {
  margin-top: 0;
  color: inherit;
}
.timeline-body > p,
.timeline-body > ul {
  margin-bottom: 0;
  padding-bottom: 0;
}
.timeline-body > p + p {
  margin-top: 0px;
}

</style>
<!--<div class="container" width: 1026px></div>-->
    <ul class="timeline">
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/3714492"  target="_blank" style="font-size:25px;">1986</a> Discovery<sup>[2]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/3714492"  target="_blank" style="font-size:25px;">1986</a> Secondary structure of type I HHR<sup>[3]</sup></h4>
            </div>
            <div class="timeline-body">
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/2436805"  target="_blank" style="font-size:25px;">1987</a> Secondary structure of type III HHR<sup>[4]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/1944278"  target="_blank" style="font-size:25px;">1991</a> Internal loops are important<sup>[5]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/7969422"  target="_blank" style="font-size:25px;">1994</a> Crystal structure of type III HHR<sup>[6]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/9818150"  target="_blank" style="font-size:25px;">1998</a> Dense positive charge is critical for catalysis<sup>[7]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/15910000"  target="_blank" style="font-size:25px;">2005</a> G8 and G12 is critical for catalysis<sup>[8]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/15910000"  target="_blank" style="font-size:25px;">2005</a> Catalytic mechanism<sup>[9]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/16859740"  target="_blank" style="font-size:25px;">2006</a> Crystal structure of type I HHR<sup>[10]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/21257745"  target="_blank" style="font-size:25px;">2011</a> Secondary structure of type II HHR<sup>[12]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/24156941"  target="_blank" style="font-size:25px;">2013</a> Profound catalytic mechanism<sup>[14]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/25981451"  target="_blank" style="font-size:25px;">2015</a> Trans-Hoogsteen greatly enhances the activity of the minimal hammer ribozyme<sup>[16]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/28448743"  target="_blank" style="font-size:25px;">2017</a> Found on some retrozyme sequences<sup>[18]</sup></h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/28530384"  target="_blank" style="font-size:25px;">2017</a> Mg2+ is critical for catalysis by activating G12<sup>[19]</sup></h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/30340787"  target="_blank" style="font-size:25px;">2018</a> Complex mechanism of enhancing activity<sup>[22]</sup></h4>
            </div>
          </div>
        </li>
        
    </ul>

</html>


## Description

***

<font size=4>The hammerhead ribozyme is an RNA motif that catalyzes reversible cleavage and ligation reactions at a specific site within an RNA molecule. It is one of several catalytic RNAs (ribozymes) known to occur in nature. It serves as a model system for research on the structure and properties of RNA, and is used for targeted RNA cleavage experiments, some with proposed therapeutic applications. Named for the resemblance of early secondary structure diagrams to a hammerhead shark, hammerhead ribozymes were originally discovered in two classes of plant virus-like RNAs: satellite RNAs and viroids. They have subsequently been found to be widely dispersed within many forms of life.</font><br>


## Structure

***

> 2D representation

<font size=4>The secondary structure of the full-length hammerhead ribozyme with the general acid (red) and base (blue) is shown. The large arrow (baby blue) points to the scissile bond. The green line notes the tertiary interaction between the terminal loop of helix II and the internal loop in helix I. </font><br>

<table><tr>
<td>
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Document</title>
</head>
<style>
   body {
    width: 100%;
    height: 100vh;
}
   button {
   margin-right: 0px;
}
   .main-container {
    display: flex;
    align-items: left;
    justify-content: center;
    height: 100%;
}
   .zoom-wrapper1 {
    width: 450px;
    height: 400px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper1">
            <div class="zoom-area1">
                <img src="https://www.ribocentre.org/images/HammerheadPic/Hammerhead2D.svg" alt="drawing" style="height:160px" />
            </div>
        </div>
    </div>
    <script src="https://www.ribocentre.org/js/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper1 = document.querySelector(".zoom-wrapper1");
      var panzoom1 = Panzoom(document.querySelector(".zoom-area1"), {
      maxScale: 6
      });
      zoomWraper1.addEventListener("wheel", panzoom1.zoomWithWheel);
      panzoom1.zoom(300 / document.querySelector(".zoom-area1 img").height);
      panzoom2.pan(0, 0);
      </script>
</td>
<td>
<link rel="stylesheet" type="text/css" href="https://www.ribocentre.org/css/fornac.css" media="screen" />

<div id="custom_colors"></div>
<form onsubmit="return handleCustomColorApply()" class="optionsform">
  <textarea id="CustomColorText" name="CustomColorText" textarea name="hide" style="display:none;" >
    1-5:#68AF31 56:blue 40:red 59-63:#68AF31 7-12:#18529A 14:#18529A 17-25:#18529A 29-34:#18529A 42-45:#C06D23 52-55:#C06D23 
  </textarea>
</form>
<meta charset="utf-8" />
    <script type="text/javascript" src="https://www.ribocentre.org/js/jquery.js"></script>
    <script type="text/javascript" src="https://www.ribocentre.org/js/d3.js"></script>
    <script type='text/javascript' src='https://www.ribocentre.org/js/demo/rsvfornac.js'></script>
    <script type="text/javascript">
      "use strict"
      function customColorsContainer() {
         let container = new fornac.FornaContainer("#custom_colors",
                 {'applyForce': 1,'editable':'true', 'initialSize':[450,400]});
         let options = {'structure': '(((((.((((((.(..(((()))))...)))))).......((((......))))...)))))',
             'sequence':             'GGCGUCCUGGUAUCCAAUCCGGAUGUACUACCAGCUGAUGAGUCCCAAAUAGGACGAAACGCC'
         };
         container.addRNA(options.structure, options);
         return container;
     }
     let cc = customColorsContainer();
 
     function handleCustomColorApply() {
       cc.addCustomColorsText(document.getElementById("CustomColorText").value);
       return false;
     }
     handleCustomColorApply();
 
     </script>

</td>
</tr></table><br>

> 3D visualisation

<font size=4>The overall structure of the hammerhead ribozyme is shown. This representation was generated from PDB ID: 3ZD5 at 2.2 Å resolution. Note the tertiary interaction between the terminal loop of helix II and the internal loop in helix I. </font>

<table><tr>
<td><img src="https://www.ribocentre.org/images/HammerheadPic/hammerhead3D.png" alt="drawing" style="weight:200px;height:350px;" border=0></td>
<td>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>PDBe Molstar</title>
    <!-- Molstar CSS & JS -->
    <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
    <script src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-1.2.1.js"></script>
        <script>
        function customize()
        {
            viewerInstance.canvas.setBgColor({r:255, g:255, b:255})
        }
        </script>

  <style>
      * {
          margin: 0;
          padding: 0;
          box-sizing: border-box;
      }
      .msp-plugin ::-webkit-scrollbar-thumb {
          background-color: #474748  !important;
      }
      .viewerSection {
        padding-top: 0px;
      }
      .controlsSection {
        width: 300px;
        float: margin-right;
        padding: 10px 0 0 0px;
        margin-right: 10px;
      }
      #myViewer{
        float:left;
        width:450px;
        height: 450px;
        position:relative;  
      }

  </style>
  </head>
  
	
  <!-- Molstar container -->
  <div id="myViewer">
  </div>
  <script>
    var viewerInstance = new PDBeMolstarPlugin();
    var options = {
      moleculeId: '3zd5',
      expanded: false,
      expanded: false,
      hideControls: true
      }
    var viewerContainer = document.getElementById('myViewer');
    viewerInstance.render(viewerContainer, options);
  </script>
</html></td>
</tr></table><br>


> Catalytic centre

Close view of the catalytic center of the hammerhead ribozyme. This image was drawn using PDB ID: 3ZD5 at 2.2 Å resolution.<br>

The hammerhead ribozyme mechanism is shown. G12 acts as the general base (blue) and G8 functions as the general acid (red).<br>

<table><tr>
<td><img src="https://www.ribocentre.org/images/HammerheadPic/hammerhead-m1.png" alt="drawing" style="width:420px;height:300px" border=1 px></td>
<td><img src="https://www.ribocentre.org/images/HammerheadPic/hammerhead-m2.png" alt="drawing" style="width:480px;height:360px" border=1 px></td>
</tr></table><br>

> Catalytic mechanism

The hammerhead ribozyme mechanism is reasonably well understood, although not totally established.  The rate of cleavage rises across the whole accessible range of pH<sup>[25]</sup>, suggesting that there are no participating groups of low pKa while substitution of G8 and G12 by 2,6-diaminopurine led to bell-shaped pH dependence<sup>[9]</sup>, pointing to a role for these nucleotides in catalysis. In the crystal structure of the extended form of the hammerhead ribozyme the nucleobase of G12 is positioned with its N1 directed toward the O2′ nucleophile. Its location thus suggests that the hammerhead is another ribozyme using N1 of a guanine nucleobase to act as the general base to remove the proton from the 2’-OH nucleophile in the cleavage reaction. This is supported by alkylation of G12 N1 using a substrate modified by 2′-bromoacetamide <sup>[26]</sup>.  In the crystal the O2′ of G8 is located adjacent to the O5′ of the scissile phosphate, well positioned to donate a proton to the oxyanion leaving group. This is consistent with a marked alteration in the rate–pH profile for an atomic mutant where the 2′ hydroxyl group of G8 was replaced by an amine<sup>[23]</sup>. The normal pKa of a 2′-hydroxyl group is 13.5. If it remains at this value in the hammerhead ribozyme, while most molecules will have a proton, it is very unlikely to be donated to another group. Therefore it seems probable that the pKa would need to be lowered, and the obvious way to achieve that would be juxtaposition of a positively-charged metal ion. QM/MM calculations suggested that a metal ion could move closer to lower the pKa of the G8 O2′ in the transition state <sup>[27]</sup>. This is supported by the restoration of cleavage activity resulting either from the absence of divalent cations or the presence of a deoxyribose at G8 by 5′-phosphorothiolate substitution <sup>[26]</sup>.  Moreover the cleavage rate exhibits a log-linear relationship as a function of the metal ion pKa <sup>[26]</sup>. In summary the hammerhead ribozyme appears to employ general acid-base catalysis using G N1 as the base, and a 2’-hydroxyl group (probably activated by a nearby metal ion) as general acid in cleavage. It is however conceivable that there is a second mechanistic channel whereby a water of hydration in the inner sphere of hydration of a bound metal ion acts as the general acid. This would then be closely similar to current ideas about the mechanism of the similar pistol ribozyme.<br><br>



## References

***

**[1] Autolytic Processing of Dimeric Plant Virus Satellite RNA.**<br>
Prody, G. A., J. T. Bakos, J. M. Buzayan, I. R. Schneider and G. Bruening <br>
[Science 231(4745): 1577-1580.(1986)](https://www.ncbi.nlm.nih.gov/pubmed/17833317)<br><br>

**[2] Self-cleavage of plus and minus RNA transcripts of avocado sunblotch viroid.**<br>
Hutchins, C. J., P. D. Rathjen, A. C. Forster and R. H. Symons <br>
[Nucleic Acids Research 14(9): 3627-3640.(1986)](https://www.ncbi.nlm.nih.gov/pubmed/3714492)<br><br>

**[3] Self-cleavage of plus and minus RNA transcripts of avocado sunblotch viroid.**<br>
Hutchins, C. J., P. D. Rathjen, A. C. Forster and R. H. Symons <br>
[Nucleic Acids Research 14(9): 3627-3640(1986)](https://www.ncbi.nlm.nih.gov/pubmed/3714492)<br><br>

**[4] Self-cleavage of plus and minus RNAs of a virusoid and a structural model for the active sites.**<br>
Forster, A. C. and R. H. Symons <br>
[Cell 49(2): 211-220.(1987)](https://www.ncbi.nlm.nih.gov/pubmed/2436805)<br><br>

**[5] Newt satellite 2 transcripts self-cleave by using an extended hammerhead structure.**<br>
Pabón-Peña, L. M., Y. Zhang and L. M. Epstein <br>
[Mol Cell Biol 11(12): 6109-6115.(1991)](https://www.ncbi.nlm.nih.gov/pubmed/1944278)<br><br>

**[6] Three-dimensional structure of a hammerhead ribozyme.**<br>
Pley, H. W., K. M. Flaherty and D. B. McKay <br>
[Nature 372(6501): 68-74.(1994)](https://www.ncbi.nlm.nih.gov/pubmed/7969422)<br><br>

**[7] The hammerhead, hairpin and VS ribozymes are catalytically proficient in monovalent cations alone.**<br>
Murray, J. B., A. A. Seyhan, N. G. Walter, J. M. Burke and W. G. Scott <br>
[Chem Biol 5(10): 587-595.(1998)](https://www.ncbi.nlm.nih.gov/pubmed/9818150)<br><br>

**[8] Model for general acid-base catalysis by the hammerhead ribozyme: pH-activity relationships of G8 and G12 variants at the putative active site.**<br>
Han, J. and J. M. Burke <br>
[Biochemistry 44(21): 7864-7870.(2005)](https://www.ncbi.nlm.nih.gov/pubmed/15910000)<br><br>

**[9] Model for general acid-base catalysis by the hammerhead ribozyme: pH-activity relationships of G8 and G12 variants at the putative active site.**<br>
Han, J. and J. M. Burke <br>
[Biochemistry 44(21): 7864-7870.(2005)](https://www.ncbi.nlm.nih.gov/pubmed/15910000)<br><br>

**[10] Tertiary contacts distant from the active site prime a ribozyme for catalysis.**<br>
Martick, M. and W. G. Scott <br>
[Cell 126(2): 309-320.(2006)](https://www.ncbi.nlm.nih.gov/pubmed/16859740)<br><br>

**[11] Capturing hammerhead ribozyme structures in action by modulating general base catalysis.**<br>
Chi, Y. I., M. Martick, M. Lares, R. Kim, W. G. Scott and S. H. Kim <br>
[PLoS Biol 6(9): e234.(2008)](https://www.ncbi.nlm.nih.gov/pubmed/18834200)<br><br>

**[12] Structure-based search reveals hammerhead ribozymes in the human microbiome.**<br>
Jimenez, R. M., E. Delwart and A. Lupták <br>
[J Biol Chem 286(10): 7737-7743.(2011)](https://www.ncbi.nlm.nih.gov/pubmed/21257745)<br><br>

**[13] Active-site monovalent cations revealed in a 1.55-Å-resolution hammerhead ribozyme structure.**<br>
Anderson, M., E. P. Schultz, M. Martick and W. G. Scott <br>
[J Mol Biol 425(20): 3790-3798.(2013)](https://www.ncbi.nlm.nih.gov/pubmed/23711504)<br><br>

**[14] Bridging the gap between theory and experiment to derive a detailed understanding of hammerhead ribozyme catalysis.**<br>
Lee, T. S., K. Y. Wong, G. M. Giambasu and D. M. York <br>
[Prog Mol Biol Transl Sci 120: 25-91.(2013)](https://www.ncbi.nlm.nih.gov/pubmed/24156941)<br><br>

**[15] Structural and catalytic effects of an invariant purine substitution in the hammerhead ribozyme: implications for the mechanism of acid-base catalysis.**<br>
Schultz, E. P., E. E. Vasquez and W. G. Scott <br>
[Acta Crystallogr D Biol Crystallogr 70(Pt 9): 2256-2263.(2014)](https://www.ncbi.nlm.nih.gov/pubmed/25195740)<br><br>

**[16] Minimal Hammerhead Ribozymes with Uncompromised Catalytic Activity.**<br>
O'Rourke, S. M., W. Estell and W. G. Scott <br>
[J Mol Biol 427(14): 2340-2347.(2015)](https://www.ncbi.nlm.nih.gov/pubmed/25981451)<br><br>

**[17] New classes of self-cleaving ribozymes revealed by comparative genomics analysis.**<br>
Weinberg, Z., P. B. Kim, T. H. Chen, S. Li, K. A. Harris, C. E. Lünse and R. R. Breaker <br>
[Nat Chem Biol 11(8): 606-610.(2015)](https://www.ncbi.nlm.nih.gov/pubmed/26167874)<br><br>

**[18] Circular RNAs with hammerhead ribozymes encoded in eukaryotic genomes: The enemy at home.**<br>
de la Peña, M. and A. Cervera <br>
[RNA Biol 14(8): 985-991.(2017)](https://www.ncbi.nlm.nih.gov/pubmed/28448743)<br><br>

**[19] Divalent Metal Ion Activation of a Guanine General Base in the Hammerhead Ribozyme: Insights from Molecular Simulations.**<br>
Chen, H., T. J. Giese, B. L. Golden and D. M. York <br>
[Biochemistry 56(24): 2985-2994.(2017)](https://www.ncbi.nlm.nih.gov/pubmed/28530384)<br><br>

**[20] Numerous small hammerhead ribozyme variants associated with Penelope-like retrotransposons cleave RNA as dimers.**<br>
Lünse, C. E., Z. Weinberg and R. R. Breaker <br>
[RNA Biol 14(11): 1499-1507.(2017)](https://www.ncbi.nlm.nih.gov/pubmed/27858507)<br><br>

**[21] Structure-based mechanistic insights into catalysis by small self-cleaving ribozymes.**<br>
Ren, A., R. Micura and D. J. Patel <br>
[Curr Opin Chem Biol 41: 71-83.(2017)](https://www.ncbi.nlm.nih.gov/pubmed/29107885)<br><br>

**[22] Structural Simplicity and Mechanistic Complexity in the Hammerhead Ribozyme.**<br>
O'Rourke, S. M. and W. G. Scott <br>
[Prog Mol Biol Transl Sci 159: 177-202.(2018)](https://www.ncbi.nlm.nih.gov/pubmed/30340787)<br><br>

**[23] Comparison of the Structures and Mechanisms of the Pistol and Hammerhead Ribozymes.**<br>
Wilson, T. J., Y. Liu, N. S. Li, Q. Dai, J. A. Piccirilli and D. M. J. Lilley <br>
[J Am Chem Soc 141(19): 7865-7875.(2019)](https://www.ncbi.nlm.nih.gov/pubmed/31017785)<br><br>

**[24] Detection of Low-Abundance Metabolites in Live Cells Using an RNA Integrator.**<br>
You, M., J. L. Litke, R. Wu and S. R. Jaffrey <br>
[Cell Chem Biol 26(4): 471-481.e473.(2019)](https://www.ncbi.nlm.nih.gov/pubmed/30773480)<br><br>

**[25] The hammerhead cleavage reaction in monovalent cations.**
Curtis, E.A. and Bartel, D.P. <br>
[RNA, 7, 546-552.(2001)(https://www.ncbi.nlm.nih.gov/pubmed/30773480)<br><br>

**[26] Probing general acid catalysis in the hammerhead ribozyme.**<br>
Thomas, J.M. and Perrin, D.M. <br>
[J. Amer. Chem. Soc., 131, 1135-1143.(2009)](https://www.ncbi.nlm.nih.gov/pubmed/30773480)<br><br>

**[27] Role of Mg2+ in hammerhead ribozyme catalysis from molecular simulation.**<br>
Lee, T.S., Silva Lopez, C., Giambasu, G.M., Martick, M., Scott, W.G. and York, D.M. <br>   
[J. Amer. Chem. Soc., 130, 3053-3064.(2008)](https://www.ncbi.nlm.nih.gov/pubmed/30773480)<br><br>

